spec/code-manager-spec.js

```diff
@@ -24,13 +24,13 @@ describe("CodeManager", () => {
 
     it("getRow", () => {
       spyOn(store.editor, "lineTextForBufferRow");
-      CM.getRow(123);
+      CM.getRow(store.editor, 123);
```

Comment:
I would prefer removing the usage of `store` from this test file. Since all are pure function this can now be done 🎉 

Either passing in `const editor = atom.workspace.buildTextEditor()` or supplying a custom mock should be fine.